app/os_windows.go

```diff
@@ -280,6 +281,8 @@ func windowProc(hwnd syscall.Handle, msg uint32, wParam, lParam uintptr) uintptr
 		w.w.Event(key.FocusEvent{Focus: true})
 	case windows.WM_KILLFOCUS:
 		w.w.Event(key.FocusEvent{Focus: false})
+	case windows.WM_NCACTIVATE:
+		w.w.Event(system.FocusEvent{Focus: wParam == windows.TRUE})
```

Comment:
On Windows, the `KILLFOCUS` happens when another window have keyboard focus, which might also happens when some child window is in focus. The `NCACTIVATE` sounds better, since it report when the window decoration should update.